src/views/virtualmachinesinstance/list/VirtualMachinesInstancesList.tsx

```diff
@@ -38,7 +38,7 @@ const VirtualMachinesInstancesList: React.FC<VirtualMachinesInstancesListProps>
 
   return (
     <>
-      <ListPageHeader title={t('Virtual Machines Instances')} />
+      <ListPageHeader title={t('VirtualMachinesInstances')} />
```

Comment:
please change this to `VirtualMachineInstances`
you can see which label to use by looking at the resource's model propertoes `label` or `labelPlural`:
https://github.com/kubevirt-ui/kubevirt-api/blob/main/console/models/VirtualMachineInstanceModel.ts#L6-L7